examples/text_classification/tc_mnli_transformers.ipynb

```diff
@@ -0,0 +1,6283 @@
+{
```

Comment:
<p>would you mind to move this list to the parameters cell? that way we can control how many models we want to use in the tests</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/434/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>